### PR TITLE
[Docs] Streams user guide

### PR DESCRIPTION
**Pull request by palmtree5**
# Enhancement request

#### Describe the enhancement

This adds a user guide for the streams cog to the docs.

#### Does this enhancement break existing functionality?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by palmtree5** [comment]
Marked ready for review.

File: docs/cog_guides/streams.rst [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+
+**Syntax**
+
+.. code-block:: none
+    
+    [p]streamset message
+
+**Description**
+
+Manage custom message for stream alerts.
```
Comment (Flame442) [inline docs/cog_guides/streams.rst]:
```suggestion
Manage custom messages for stream alerts.
```
OR
```suggestion
Manage the custom message for stream alerts.
```

**Review by Flame442 — DISMISSED** [review]
Almost all set!

File: docs/cog_guides/streams.rst (hunk 1 above)
Comment (palmtree5) [reply docs/cog_guides/streams.rst]: Fixed, though it's worth noting that was basically a copy/paste from the command's docstring so I took the liberty of fixing it there too

**Review by Flame442 — DISMISSED** [review]
Thanks!

**Review by Flame442 — APPROVED** [review]
*grumbles*

**Pull request merged**